es6-promises/index.js

```diff
@@ -0,0 +1,9 @@
+const takeAChance = require('./take-a-chance');
+
+const answer = takeAChance('michael');
+answer.then(value => {
+  console.log(value);
+});
+answer.catch(error => {
+  console.error(error);
+});
```

Comment:
Your rejection handler should only output the error message, not the whole error object:

---

![image](https://user-images.githubusercontent.com/16143771/180912888-358dc66a-0cdd-4b12-9289-3b8c7d526928.png)

---

![image](https://user-images.githubusercontent.com/16143771/180912918-7af2ff02-9641-4c91-a597-8c3466d81c97.png)